src/hooks/useWithdrawLiquidityHandler.tsx

```diff
@@ -113,11 +111,11 @@ export const useWithdrawLiquidityHandler = () => {
           setPosManager(new ethers.Contract(address, PosManager.abi, provider))
         }
       } else {
-        if (accountInfo && accountInfo?.address) {
-          setDepPool(new ethers.Contract(address, DepPool.abi, provider.getSigner(accountInfo?.address)))
-        } else {
-          setDepPool(new ethers.Contract(address, DepPool.abi, provider))
-        }
+        // if (accountInfo && accountInfo?.address) {
+        //   setDepPool(new ethers.Contract(address, DepPool.abi, provider.getSigner(accountInfo?.address)))
+        // } else {
+        //   setDepPool(new ethers.Contract(address, DepPool.abi, provider))
+        // }
```

Comment:
same with this

Reply:
DepPool was a legacy object that connected to the hackathon abis. There was no close equivalent in the v1-periphery or v1-core. So I didn't know what to do with it. I commented it out to be compliant with the new packages and removing of v0-hackathan abis. This function's backend was never connected to the v1 contracts yet, so this commented code will be fixed when the connection is being worked on here To be connected again with this task: https://www.notion.so/S4-10-03-10-14-39a87ba77fc4445287762e021b523b30?p=3df17d65f1654441801699ed253ad154&pm=s.